### PR TITLE
Fixed README.md rendering

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Fixed README.md rendering

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,20 +304,17 @@ OpenShift LightSpeed (OLS) is an AI powered assistant that runs on OpenShift and
...
+```
 
-      > This action may be required for self-hosted LLMs.
+ > This action may be required for self-hosted LLMs.
 
 ## 12. Registering new LLM provider
-    Please look info this
-    [contributing guide chapter](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel)
-    for more info.
+   Please look info this [contributing guide chapter](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
```
Comment (onmete) [inline README.md:317]: Maybe change it for:
Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.

Comment (tisnik) [reply README.md]: @onmete updated accordingly

**Pull request merged**